packages/celotool/src/cmds/geth/simulate_client.ts

```diff
@@ -51,30 +51,53 @@ export const builder = () => {
       description: 'Mnemonic used to generate account addresses',
       demand: 'A mnemonic must be provided',
     })
+    .options('client-count', {
+      type: 'number',
+      description: 'Number of clients to simulate',
+      default: 1,
+    })
+    .options('reuse-client', {
+      type: 'boolean',
+      description: 'Use the same light client for all the threads/accounts',
```

Comment:
description should be changed, since I think with reuse-client true it uses a full node rather than a light client.

Reply:
Updated. Makes sense to allow the syncmode to be configured as a `.env` or a cmd option in the loadtests?